applications/incompressible_navier_stokes/periodic_hill/include/postprocessor.h

```diff
@@ -104,7 +104,13 @@ class MyPostProcessor : public PostProcessor<dim, Number>
     }
 
     // line plot statistics
-    line_plot_calculator_statistics->evaluate(velocity, pressure, time, time_step_number);
+    if(line_plot_calculator_statistics->time_control_statistics.needs_evaluation(time,
+                                                                                 time_step_number))
+      line_plot_calculator_statistics->evaluate(
```

Comment:
I such cases I would introduce brackets since its a bit hard to read in my opinion. Just as a minor comment for the future. No need to realize this here before merging.

Reply:
I prefer not to care about this lateron:
[0989d33](https://github.com/exadg/exadg/pull/262/commits/0989d3386318e0be33f226c31c569eb4519196af)